### PR TITLE
Attach env if not meta is defined

### PR DESCRIPTION
**Pull request by d6u**
@danpaz Fix a case when no meta is provided, no env is also attached.

/cc @ahaurw01

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -133,7 +133,11 @@ function _getSentryMeta(meta) {
   var extra;
 
   if (!meta) {
-    return null;
+    return {
+      tags: {
+        env: process.env.NODE_ENV || 'development'
+      }
+    };
```
Comment (danpaz) [inline index.js]: Haven't looked super closely yet, but wouldn't `_.defaults` or `_.set` handle most of this logic?

**Comment by d6u** [comment]
Good idea! I replaced logic with `merge`.

**Comment by d6u** [comment]
Updated

File: index.js [hunk 2]
```diff
@@ -127,39 +127,18 @@ function _logger(level) {
 }
 
 function _getSentryMeta(meta) {
```
Comment (danpaz) [inline index.js:142]: Can you add a jsdoc for this method?

**Comment by danpaz** [comment]
Looks great! One comment to add jsdoc, then Approved.

**Comment by d6u** [comment]
I almost want to make jsdoc a requirement in eslint. Hard to remember.

**Pull request merged**